lib/index.js

```diff
@@ -38,9 +38,15 @@ exports.register = function (server, options, next) {
     apply: true
   })
 
-  server.decorate('server', [options.hapiDecoratePrefix || '', 'action'].join(''), internals.action(server))
-  server.decorate('reply', [options.hapiDecoratePrefix || '', 'act'].join(''), internals.replyAct)
-  server.handler([options.hapiDecoratePrefix || '', 'act'].join(''), internals.handlerAct)
+  if (options.decoratePrefix) {
+    server.decorate('server', `${options.decoratePrefix}_action`, internals.action(server))
+    server.decorate('reply', `${options.decoratePrefix}_act`, internals.replyAct)
+    server.handler(`${options.decoratePrefix}_act`, internals.handlerAct)
```

Comment:
I wouldn't prefix it with a `_` but ensure camelCase e.g

```
decoratePrefix = 'test'
testAction()
testAct()
testAdd()
```
the underscore was just an example. Thanks